recipe/meta.yaml

```diff
@@ -22,11 +22,12 @@ requirements:
     - setuptools
   run:
     - python >={{ python_min }}
+    - nvidia-ml-py ~={{ major_version }}.0
```

Comment:
This is equivalent to `nvidia-ml-py >12.0,<13.0a`

More details in the [package match specification docs]( https://docs.conda.io/projects/conda-build/en/stable/resources/package-spec.html#package-match-specifications )